ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -41,6 +42,7 @@
     }
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LinearRao.class);
+    private boolean useFallbackSensiParams = false;
```

Comment:
That's not really clean to have this non-static field initialize outside a constructor. I'd rather use it as a local variable in the run method. It would have to be passed as argument in several methods but I think it would be cleaner.

Reply:
Do you mean that I should pass this boolean by reference to my private methods?

Reply:
thing is it is not possible to pass primitive types by reference in java... You can try with AtomicBoolean class